docs/data/base/components/tooltip/tooltip.md

```diff
@@ -1,14 +1,389 @@
+
+```jsx
+<Tooltip.Positioner side="right" alignment="start">
+  <Tooltip.Popup>Tooltip</Tooltip.Popup>
+</Tooltip.Positioner>
+```
+
+Due to collision detection, the tooltip may change its placement to avoid overflow. Therefore, your explicitly specified `side` and `alignment` props act as "ideal", or preferred, values.
+
+To access the true rendered values, which may change as the result of a collision, the content element receives data attributes:
+
+```jsx
+// Rendered HTML (simplified)
+<div>
+  <div data-side="left" data-alignment="end">
+    Tooltip
+  </div>
+</div>
+```
+
+This allows you to conditionally style the tooltip based on its rendered side or alignment.
+
+## Offset
+
+The `sideOffset` prop creates a gap between the anchor and tooltip popup, while `alignmentOffset` slides the tooltip popup from its alignment, acting logically for `start` and `end` alignments.
+
+```jsx
+<Tooltip.Positioner sideOffset={10} alignmentOffset={10}>
+```
+
+## Delay
+
+To change how long the tooltip waits until it opens or closes, use the `delay` and `closeDelay` props, which represent how long the tooltip waits after the cursor rests on the trigger to open, or moves away from the trigger to close, in milliseconds:
+
+```jsx
+<Tooltip.Root delay={200} closeDelay={200}>
+```
+
+The delay type can be changed from `"rest"` (user's cursor is static over the trigger for the given timeout in milliseconds) to `"hover"` (the user's cursor has entered the trigger):
+
+```jsx
+<Tooltip.Root delayType="hover">
+```
+
+## Controlled
+
+To control the tooltip with external state, use the `open` and `onOpenChange` props:
+
+```jsx
+function App() {
+  const [open, setOpen] = React.useState(false);
+  return (
+    <Tooltip.Root open={open} onOpenChange={setOpen}>
+      {/* Subcomponents */}
+    </Tooltip.Root>
+  );
+}
+```
+
+## Arrow
+
+To add an arrow (caret or triangle) inside the tooltip content that points toward the center of the anchor element, use the `Tooltip.Arrow` component:
+
+```js
+<Tooltip.Positioner>
+  <Tooltip.Popup>
+    Tooltip
+    <Tooltip.Arrow />
+  </Tooltip.Popup>
+</Tooltip.Positioner>
+```
+
+It automatically positions a wrapper element that can be styled or contain a custom SVG shape.
+
+## Cursor following
+
+The tooltip can follow the cursor on both axes or one axis using the `followCursorAxis` prop on `Tooltip.Root`. Possible values are: `none` (default), `both`, `x`, or `y`.
+
+{{"demo": "UnstyledTooltipFollowCursor.js"}}
+
+## Anchoring
+
+By default, the `Trigger` acts as the anchor. This can be changed to another element, either virtual or real:
+
+```js
+<Tooltip.Positioner
+  // Element
+  anchor={domElement}
+  // React ref
+  anchor={anchorRef}
+  // VirtualElement
+  anchor={{
+    getBoundingClientRect: () => DOMRect,
+    contextElement: domElement, // optional
+  }}
+>
```

Comment:
I understand you aimed for a concise example but I'm afraid this could confuse beginners making them think you can set all of these at the same time. Could you split these examples?